### PR TITLE
feat(configuration): read only configuration

### PR DESCRIPTION
**Pull request by rodrigobasilio2022**
This PR is related to issue IDC #3357 , that want to prevent editing in OHIF. It creates a new variable that prevents the edition in Segmentation table (SEG and RTSTRUCT) and open measurements with locking enabled on creation

### CODE REVIEW

**Comment by sedghi** [comment]
You should take out the autoLoad mode from this PR and only keep the pieces that is required to be changed in the core.

**Comment by rodrigobasilio2022** [comment]
Already done that

File: extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const SR_SOPCLASSHANDLERID =
  * @param {*} param0
  */
 function TrackedMeasurementsContextProvider(
-  { servicesManager, commandsManager, extensionManager }, // Bound by consumer
+  { servicesManager, commandsManager, extensionManager, appConfig }, // Bound by consumer
```
Comment (sedghi) [inline extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx:29]: You don't need to pass the appConfig as props in most of the changed files. Take a look at how we use `useAppConfig` hook

Comment (rodrigobasilio2022) [reply extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx]: Changed as requested

File: platform/app/public/config/default.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 window.config = {
+  disableEditing: true,
```
Comment (sedghi) [inline platform/app/public/config/default.js:2]: can you remove this please

Comment (sedghi) [reply platform/app/public/config/default.js:2]: we should add it to the documentation in the configuration though

**Review by sedghi — CHANGES_REQUESTED** [review]
see my comments

File: platform/app/public/config/default.js (hunk 2 above)
Comment (rodrigobasilio2022) [reply platform/app/public/config/default.js]: Ok

File: platform/app/public/config/default.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 window.config = {
-  disableEditing: true,
+  // This variable disables editing in OHIF, hiding edit buttons in segmentation
+  // panel and locking already stored measurements.
+  //disableEditing: true,
```
Comment (sedghi) [inline platform/app/public/config/default.js:4]: I rather have this inside documentation and not here at all

**Review by sedghi — APPROVED** [review]
I think this is good to go. Please add your comments @igoroctaviano @wayfarer3130

**Review by igoroctaviano — APPROVED** [review]
LGTM

**Pull request merged**